Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -1234,6 +1234,11 @@ bool SkylanderPortal::CreateSkylander(const std::string& file_path, u16 sky_id,
   return true;
 }
 
+bool SkylanderPortal::RemoveSkylanderiOS(u8 sky_num)
```

Comment:
Is there a reason why `RemoveSkylanderiOS` and `LoadSkylanderiOS` exist? Can't you just call `RemoveSkylander` and `LoadSkylander` directly? I'm probably missing something here.

Reply:
I wish I could explain this, but this was the only way that it actually worked - almost as if the objective c++ had a reference to a different system instance than the C++ lib, calling it directly had no effect on the game that was displayed